include/LightGBM/c_api.h

```diff
@@ -988,6 +988,24 @@ LIGHTGBM_C_EXPORT int LGBM_BoosterFeatureImportance(BoosterHandle handle,
                                                     int importance_type,
                                                     double* out_results);
 
+/*!
+ * \brief Get model upper bound value.
+ * \param handle Handle of booster
+ * \param[out] out_results Result pointer to max value
+ * \return 0 when succeed, -1 when failure happens
+ */
+LIGHTGBM_C_EXPORT int LGBM_BoosterGetUpperBoundValue(BoosterHandle handle,
```

Comment:
Can you please add corresponding methods for Python and R wrappers somewhere around here
https://github.com/microsoft/LightGBM/blob/dbb804f2350f280f3c927f792f7e55ac9c021707/python-package/lightgbm/basic.py#L2163-L2203
and here
https://github.com/microsoft/LightGBM/blob/dbb804f2350f280f3c927f792f7e55ac9c021707/R-package/R/lgb.Booster.R#L312-L322
https://github.com/microsoft/LightGBM/blob/dbb804f2350f280f3c927f792f7e55ac9c021707/src/lightgbm_R.cpp#L415-L423
respectively?

Also, a simple test will be really appreciated 🙂 .
https://github.com/microsoft/LightGBM/blob/dbb804f2350f280f3c927f792f7e55ac9c021707/tests/python_package_test/test_basic.py#L39-L41